other/fizzbuzz/fizzbuzz.jl

```diff
@@ -49,6 +50,6 @@ end
 
 # Training
 for e in 0:1000
-    train!(loss, [(X, y)], opt)
+    train!(loss, ps, [(X, y)], opt)
```

Comment:
In some of these simple cases I think we should just inline `ps`.

```suggestion
    train!(loss, params(m), [(X, y)], opt)
```

Reply:
Got it